lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -127,11 +127,16 @@ def update_to_cocoapods_0_37_1
         # @todo   This can be removed for CocoaPods 1.0
         #
         def update_to_cocoapods_0_39
-          requires_update = native_targets_to_embed_in.any? do |target|
-            !target.shell_script_build_phases.find { |bp| bp.name == 'Embed Pods Frameworks' }
+          targets_to_embed = native_targets.select do |target|
+            EMBED_FRAMEWORK_TARGET_TYPES.include?(target.symbol_type)
+          end
+          requires_update = targets_to_embed.any? do |target|
+            !target.shell_script_build_phases.find { |bp| bp.name == EMBED_FRAMEWORK_PHASE_NAME }
           end
           if requires_update
-            add_embed_frameworks_script_phase
+            targets_to_embed.each do |native_target|
```

Comment:
that's fine, we don't support having them being anything but invoking the embed frameworks script to being with